firmware/src/main.cpp

```diff
@@ -69,16 +69,6 @@ void setup() {
   TJpgDec.setSwapBytes(true); // JPEG rendering setup
   TJpgDec.setCallback(tft_output);
 
-#ifdef GC9A01_DRIVER
-  Serial.println("GC9A01 Driver");
-#endif
-#ifdef ILI9341_DRIVER
-  Serial.println("ILI9341 Driver");
-#endif
-#if HARDWARE == WOKWI
```

Comment:
I'd keep the Wokwi in here, there is a PR that re-adds some of the Wokwi files.

Reply:
fixed